network-api/networkapi/templates/fragments/nav-links.html

```diff
@@ -0,0 +1,7 @@
+{% load primary_active_nav %}
+
+{% for item in menu_items %}
+  {{ pre }}<a class="{% primary_active_nav request menu_root.full_url item.full_url %}" href="{{ item.url }}" title="{{ menu_root.full_url }}, {{ item.full_url }}, {{ request.build_absolute_uri }}">{{ item.title|title }}</a>{{ post }}
```

Comment:
`full_url` is used for code comparison purposes because `.url` is not stable (it might or might not yield a relative or absolute path). However, for the actual `href` attribute it doesn't matter whether the url is relative or absolute.